RxTheme/Classes/Extensions/UIStepper+Rx.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Created by Cruz on 25/08/2018.
+//  2018 Copyright (c) RxSwiftCommunity. All rights reserved.
+//
+
+#if os(iOS)
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+
+public extension Reactive where Base: UIStepper {
+    
+    /// Bindable sink for `tintColor` property
+    public var tintColor: Binder<UIColor?> {
```

Comment:
@cruisediary Thank you for applying my review. 

But basically I do not know why `tintColor` should be added. You can use `tintColor` in the current version because [UIView+Rx](https://github.com/RxSwiftCommunity/RxTheme/blob/master/RxTheme/Classes/Extensions/UIView%2BRx.swift) already has `tintColor`.

This is my code and works fine in the current version.
```swift
themeService.rx
    .bind({ $0.tintColor }, to: stepper.rx.tintColor)
    .disposed(by: disposeBag)
```

Reply:
@cozzin 
[UIView+Rx](https://github.com/RxSwiftCommunity/RxTheme/blob/master/RxTheme/Classes/Extensions/UIView%2BRx.swift) does not support `UIStepper`'s `tintColor` because it does not inherit `UIView`, inherit `UIControl` similar to `UIButton`

Reply:
But `UIControl` does inherit `UIView` 😢 then makes sense 
https://developer.apple.com/documentation/uikit/uicontrol

Reply:
Thank you :)